examples/acceptFromIDs.js

```diff
@@ -1,6 +1,16 @@
-var SteamUser = require('steam-user');
-var client = new SteamUser();
-var TradeOfferManager = require('steam-tradeoffer-manager');
+const SteamUser = require('steam-user'),
```

Comment:
Sorry if it seems like I'm being a code Nazi (I am) – I just think the examples should look similar so newcomers can easily understand what's going on without 20 different styles being used.